code/style.css

```diff
@@ -159,6 +159,10 @@ button {
   height: 100vh;
 }
 
+.show-win-or-lose-section {
+  display: flex;
+}
+
```

Comment:
I don´t know wish way is the best, but I (or rather Isabella how helped me) put this directtielly in the script instead. Like this:

const start = () => { winOrLose.style.display = 'none' board.style.display ='flex'